libs/native/jna/src/main/java/org/elasticsearch/nativeaccess/jna/JnaNativeLibraryProvider.java

```diff
@@ -45,6 +51,11 @@ public JnaNativeLibraryProvider() {
         );
     }
 
+    @SuppressForbidden(reason = "jna library path must be set for load library to work with our own libs")
+    private static void setJnaLibraryPath() {
+        System.setProperty("jna.library.path", LoaderHelper.platformLibDir.toString());
```

Comment:
Would it be better to do this in `SystemJvmOptions` rather than in a static initializer?

Reply:
No, that was the problem we had before, it made NativeAccess unusable in the CLI. I moved it into here so that this can work even when we haven't gone through SystemJvmOptions (which is only for the main ES process).

Reply:
Got it. That makes sense.